### PR TITLE
Milestone_2: Components

### PR DESCRIPTION
**Pull request by kubilaycaglayan**
Dear TSE :wave: 

I've added the basic components to my project.

- App
- Display
- Button Group
- Button

With all my respect... :pray:

### CODE REVIEW

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default function Display(props) {
+  const props2 = { result: '0', ...props };
+  return (
+    <div>
+      { props2.result }
```
Comment (Forison) [inline src/components/Display.js:7]:
```
Display.defaultProps = {
  result: '0',
}; 
```

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default function Button(props) {
+  const props2 = { ...props };
+  return (
+    <button type="button">
```
Comment (Forison) [inline src/components/Button.js:6]: - Let us also validate this prop as well.
```
Button.propTypes = {
  name: PropTypes.string.isRequired,
};
```

File: src/components/Button.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default function Button(props) {
```
Comment (Forison) [inline src/components/Button.js:3]: Let us destructure the value of the incoming props these ways:
1. ``` export default function Button({name}) { ```
or 
2. ``` const {name} = props ;```
> this is a signal that there should be a name property passed from App.js on Button

**Comment by kubilaycaglayan** [comment]
Thanks @Forison :wave:  :green_heart:

**Pull request merged**